firmware/kingst-la/sigrok-fwextract-kingst-la2016.1

```diff
@@ -61,4 +63,5 @@ Vendor website
 .RS 4
 .RB http://www.qdkingst.com/download
 .br
-\%MD5 of v3.4.0: ca407133cb83b700983d2b704a4255c2
+\%MD5 of v3.5.0: 812bbd37a16d315a489ca33ea2868a36
+
```

Comment:
Add 3.5.1 here? I recall it was mentioned somewhere.

Reply:
I was deliberately ignoring minor versions, I see there is 3.5.2 and 3.5.3 now.

Reply:
Why? Because it is too much work? Because I think it would be helpful for users.

Reply:
I submitted this PR some five months ago, in that time there has been 2 or 3 minor software updates by the vendor. IMHO, the sigrok pull-rate would never keep up with the rate of vendor 'minor' software releases anyway.

Reply:
That's true, we won't keep up. But maybe we could add the known versions at the time we update this file. It wouldn't be up-to-date, but it would be less outdated than just major versions. Anyhow, I think it would be nice to have. If it is too much of a burden for you, we can also skip it for now.